installer/IL15_emulated_checks_init.sh

```diff
@@ -110,16 +110,18 @@ IL15_emulated_checks_init() {
       if ! grep -q "/tmp/routersploit.log" rsf.py; then
         sed -i 's/routersploit\.log/\/tmp\/routersploit\.log/' ./rsf.py
       fi
+      # patch some code to write the history files to a subdirectory in /root
+      # shellcheck disable=SC2088
+      if grep -q  '~/.history' ./routersploit/interpreter.py; then
+        sed -i 's/~\/\.history/~\/\.routersploit\/\.history/' ./routersploit/interpreter.py
```

Comment:
```suggestion
        sed -i 's/~\/\.history/\/root\/\.routersploit\/\.history/' ./routersploit/interpreter.py
```

Reply:
lets leave this with ~ for teh future ;)